README.md

```diff
@@ -330,3 +330,40 @@ To run Prometheus locally without Helm, you need to make sure Prometheus is inst
 To run Grafana locally, just download it and it will continually run on your localhost on port 3000 (use a browser to go to 127.0.0.1:3000).
 Then configure a data source to the port Prometheus is running (by default, localhost:9093) and importing the dashboard JSON located in the path *grafana-dashboards/dashbard.json*.
 
+## Additional Use Case
+Navigate to the directory istio, use the `kubectl apply -f` command to apply these configuration files.
```

Comment:
Very clear instructions! Can you just make this a subsection under "How to: Istio Deployment" since it's an extension to it. Also, it may be more informative to name it "Additional Use Case: Egress for External Requests" for example.